Fix ATS testScreenshot failed

harvan opened the pull request:
Fixed: Memory crash occurs when the screencap command is executed. 
Ported from https://github.com/projectceladon/minigbm/pull/206

Tracked-On: OAM-128783

feijiang1 reviewed (approved):
LGTM

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

Commit:
Fix ATS testScreenshot failed

Fixed: Memory crash occurs when the screencap command is executed.
Ported from https://github.com/projectceladon/minigbm/pull/206

Tracked-On: OAM-128783
Signed-off-by: Weifeng Liu <weifeng.liu@intel.com>
Signed-off-by: Wan, Hao <haox.wan@intel.com>

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon-merge/1109
